scripts/deploy.ts

```diff
@@ -92,9 +92,9 @@ async function main() {
   console.log('Contract address:', address)
   console.log(
     'Etherscan URL:',
-    `https://${
-      chainName !== 'mainnet' ? `${chainName}.` : ''
-    }etherscan.io/address/${address}`
+    `https://${chainName !== 'mainnet' ? `${chainName}.` : ''}${
+      chainId === 80001 ? 'polygonscan.com' : 'etherscan.io'
```

Comment:
I think it won't work if we use `polygon mainnet`, will it?

Reply:
Probably, but we may need paymaster on polygon as well, so don't change this